fil.js

```diff
@@ -531,23 +531,23 @@ windows.controls.specFree I-toggle ang Free Cam
 windows.controls.specObj I-toggle ang Objective Cam
 windows.controls.specFirst I-toggle ang First Person
 windows.controls.specNames I-toggle ang mga pangalan
-windows.controls.specFocus I-toggle and pokus sa isang manlalaro
+windows.controls.specFocus I-toggle ang pokus sa isang manlalaro
 windows.controls.hidePlayers Itago ang mga manlalaro
-windows.controls.streak0 Streak 1
-windows.controls.streak1 Streak 2
-windows.controls.streak2 Streak 3
-windows.controls.streak3 Streak 4
-windows.controls.streak4 Streak 5
-windows.controls.taunt0 Taunt (Psst)
-windows.controls.taunt1 Taunt (Whistle)
-windows.controls.taunt2 Taunt (Fart)
-windows.controls.taunt3 Taunt (Meme)
-windows.controls.taunt4 Taunt (WTF is This?)
-windows.controls.taunt5 Taunt (Right Now!)
+windows.controls.streak0 Unang Streak
+windows.controls.streak1 Pangalawang Streak
+windows.controls.streak2 Pangatlong Streak
+windows.controls.streak3 Pangapat na Streak
+windows.controls.streak4 Panlimang Streak
+windows.controls.taunt0 Panukso (Psst)
+windows.controls.taunt1 Panukso (Sipol)
+windows.controls.taunt2 Panukso (Utot)
+windows.controls.taunt3 Panukso (Meme)
+windows.controls.taunt4 Panukso (WTF ano 'to?)
+windows.controls.taunt5 Panukso (Ngayon na!!)
 windows.controls.prop Piliin ang Prop (Hold M1)
-windows.controls.propRand Select Random Prop
-windows.controls.propRot Toggle Rotation
-windows.controls.propRotR Reset Rotation
+windows.controls.propRand Piliin ang kahit anong prop
```

Comment:
Can we make this: "Pumili ng kahit anong prop"

Reply:
Sure, thanks.